go.mod

```diff
@@ -48,10 +49,29 @@ require (
 	k8s.io/apimachinery v0.17.9
 	k8s.io/client-go v0.17.9
 	k8s.io/code-generator v0.17.9
-	k8s.io/kubernetes v1.11.1
+	k8s.io/kubernetes v1.16.0-alpha.0.0.20190623232353-8c3b7d7679cc
 	sigs.k8s.io/controller-runtime v0.5.11
 )
 
-replace github.com/go-openapi/validate => github.com/go-openapi/validate v0.17.2
+replace (
```

Comment:
Can you briefly explain why we added many replace directives here?

Reply:
We dont needed them all actually, I had some issues during development that seems to not be an issue any longer.

Reply:
I had issues with the kuberentes packages before hand but after they where upgrade to v0.17.9 i actually thin there replace did not do much.